integrations/google_ai/README.md

```diff
@@ -3,19 +3,12 @@
 [![PyPI - Version](https://img.shields.io/pypi/v/google-ai-haystack.svg)](https://pypi.org/project/google-ai-haystack)
 [![PyPI - Python Version](https://img.shields.io/pypi/pyversions/google-ai-haystack.svg)](https://pypi.org/project/google-ai-haystack)
 
----
-
-**Table of Contents**
-
-- [Installation](#installation)
-- [License](#license)
+- [Integration page](https://haystack.deepset.ai/integrations/google-ai)
+- [Changelog](https://github.com/deepset-ai/haystack-core-integrations/blob/main/integrations/google_ai/CHANGELOG.md)
 
-## Installation
-
-```console
-pip install google-ai-haystack
-```
+---
+## Contributing
 
-## License
+Refer to the general [Contribution Guidelines](https://github.com/deepset-ai/haystack-core-integrations/blob/main/CONTRIBUTING.md).
 
-`google-ai-haystack` is distributed under the terms of the [Apache-2.0](https://spdx.org/licenses/Apache-2.0.html) license.
+To run integration tests locally, you need to export the `GOOGLE_API_KEY` environment variable.
```

Comment:
Do we need to add a line like this to all our Readmes as well? I noticed that a similar comment is needed for the Github integration readme

Reply:
ahh okay, all good then!